CMakeLists.txt

```
@@ -64,8 +63,8 @@ install(
 )
 
 if(BUILD_TESTING)
-  #  find_package(launch_testing_ament_cmake)
-  #  add_launch_test(test/system.test.py)
+  find_package(launch_testing_ament_cmake)
+  add_launch_test(test/test_joystick_relay.py)
```

Let's try enabling this too:

```suggestion
  add_launch_test(test/test_joystick_relay.py)
  add_launch_test(test/system.test.py)
```